tests/sub_group/sub_group_semantics.cpp

```diff
@@ -120,10 +120,10 @@ TEST_CASE("sub_group by-value semantics", "[sub_group]") {
 TEST_CASE("Check sycl::sub_group equality", "[sub_group]") {
   size_t code_count =
       to_integral(common_by_value_semantics::current_check::size);
-  bool result[code_count];
-  std::fill(result, result + code_count, false);
+  // Avoid the std::vector<bool>
+  std::vector<char> result(code_count, false);
   {
-    sycl::buffer<bool, 1> res_buf(result, sycl::range(code_count));
+    sycl::buffer<char, 1> res_buf(result);
```

Comment:
@bader Oops! Good catch.

Reply:
> @bader Oops! Good catch.

It was CI, not me. ;)

Reply:
I was puzzled by AdaptiveCpp passing.

Reply:
AdaptiveCpp doesn't not compile this test case: https://github.com/keryell/SYCL-CTS/blob/9c269373f6ce38efd35f7d2db6e2675c1107f83c/tests/sub_group/sub_group_semantics.cpp#L118-L120